app/services/config_downloader.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+# Service for dowloading SSG configuration files
+class ConfigDownloader
+  class << self
+    SSG_DS_FILE = File.new('config/supported_ssg.yaml')
+
+    def ssg_ds
+      File.read(SSG_DS_FILE)
+    end
+
+    def ssg_ds_checksum
+      Digest::MD5.file(SSG_DS_FILE).hexdigest
```

Comment:
Observation: MD5 can be flagged by some scanners. Suggestion is to use SHA256+.

Reply:
I only used MD5 for performance reasons. Do you really think this is some kind of security issue?

Reply:
For our purpose MD5 here is okay. The nuisance will be when some scanner starts to report this.